### ISSUE TITLE
Manual parameter names

### ISSUE THREAD

**Issue by boulis** [boulis/Castalia#6]
Reported by InHwan KIM <hwanud@gmail.com> in the forum.

The recent manual from the website has a typo when describing the use of collision model. 
It says a user has to set the parameter like the below: 

.... setting the parameter SN.node[*].communication.Radio.collisionModel  .... (and goes on) ... pg. 64

However, it is wrong and does not work, because the statement is case sensitive. 
I also tried to found why the setting does not work through this forum, but some of those answers used lower case for "communication". 
Of course, the first use of that parameter is correctly specified by using upper case for the "Communication" on page 19.
So someone's code is working and the others' not working.

You can find the reason from the ned file for the node (which can be found at src/node/Node.ned).
Since it defines the module as "Communication" by using upper case, we should follow the rule to make it correctly work.

To sum up, if you want to use collisionModel parameter of Communication.Radio submodule, 
please check "Communication" is written upper case!

**Comment by boulis** [comment]
Corrections made in the manual with the 5241f75b28d741c3cdc88b3b3f235080d1b0dbc2 commit close the issue
